docs/querying-data.md

```diff
@@ -79,6 +79,19 @@ query Posts {
 }
 ```
 
+#### Sort a collection by multiple fields and different ordering
+
+```graphql
+query Posts {
+  allPost(sort: [{ by: "date", order: DESC }, { by: "title", order: DESC }]) {
```

Comment:
```suggestion
  allPost(sort: [{ by: "date", order: DESC }, { by: "title", order: ASC }]) {
```
Just to show both order types. And the query example misses the ending ` ``` `.

Reply:
thanks o/